src/components/Sum.jsx

```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = ({ a = 0, b = 0 }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
+);
+
+export default Sum;
```

Comment:
you dont have to export `default` when you import it with { }, that;s why checks fail